src/pmp/visualization/MatCapShader.h

```diff
@@ -0,0 +1,63 @@
+//=============================================================================
+// Copyright (C) 2011-2018 The pmp-library developers
```

Comment:
Copyright should be bumped to 2020

Reply:
for new files or for changed files?

Reply:
Should be updated to current year on every change.

Reply:
updated for all files that changed